### PR TITLE
 Deploy es indexer to kubernetes

### PR DESCRIPTION
**Pull request by alahwa**

### CODE REVIEW

File: bigquery/README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ if you haven't done so already.
     docker run -p 9200:9200 docker.elastic.co/elasticsearch/elasticsearch-oss:6.2.2
     ```
```
Comment (melissachang) [inline bigquery/README.md]: Let's put deploy stuff in bigquery/deploy. This includes README.md and yaml.

File: bigquery/indexer.py [hunk 2]
```diff
@@ -18,135 +18,144 @@
 
 # Log to stderr.
 logging.basicConfig(level=logging.INFO,
```
Comment (melissachang) [inline bigquery/indexer.py]: Can you create a separate PR with formatting changes to this file, and merge into master before this PR? These diffs are hard to read.

File: bigquery/README.md (hunk 1 above)
Comment (alahwa) [reply bigquery/README.md]: Done

File: bigquery/indexer.py (hunk 2 above)
Comment (alahwa) [reply bigquery/indexer.py]: Done

**Pull request closed without merging**